smart_canvas/background.py

```diff
@@ -1,20 +1,18 @@
 import cv2
 import numpy as np
-
+import mediapipe as mp
+mp_drawing = mp.solutions.drawing_utils
+mp_selfie_segmentation = mp.solutions.selfie_segmentation
 
 class ForegroundMask:
     """
     Class that gives the foreground mask.
     """
 
     def __init__(self):
-        self.back_sub = cv2.createBackgroundSubtractorMOG2(
-            history=100, varThreshold=30, detectShadows=False)
-        self.fg_mask = None
-        self.frame_list = []
-
-    def create_background(self, frame):
-        self.frame_list.append(frame)
+        self.selfie_segmentation = mp_selfie_segmentation.SelfieSegmentation(model_selection=1)
+        self.bg_image = None
+        self.output_image = None
 
     def fill_holes(self, mask):
```

Comment:
Can we remove fill_holes?
There seems to be no usage of it.

Reply:
I suppose, I left it there for if we want to keep in for the future but that is not part of the philosophy of this. Will be removed.